### PR TITLE
Expose dcmo:fields in DCModel class

### PR DESCRIPTION
**Pull request by geraldgole**

### CODE REVIEW

File: src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java [hunk 1]
```diff
@@ -95,6 +106,9 @@ public int hashCode() {
         @JsonProperty("dcmf:name")
         private String name;
 
+        @JsonProperty("dcmf:documentation")
+        private String description;
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java]: it's clearer to keep the same naming : documentation

Comment (geraldgole) [reply src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java]: OK

File: src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java [hunk 2]
```diff
@@ -25,6 +25,9 @@
     @JsonProperty("dcmo:globalFields")
     private List<DcModelField> fields;
 
+    @JsonProperty("dcmo:fields")
+    private List<DcModelField> dcFields;
```
[the remark this answers is not in the document]
Comment (geraldgole) [reply src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java]: I thought you wanted to keep actual property and getter/setter for compatibility.
But obviously, I strongly prefer `fields` and `globalFields`

I change that

[the remark this answers is not in the document]
Comment (geraldgole) [reply src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java]: Pushed

**Pull request merged**